geomstats/visualization.py

```diff
@@ -98,6 +98,46 @@ def draw_points(self, ax, **scatter_kwargs):
         points_z = np.vstack([point[2] for point in self.points])
         ax.scatter(points_x, points_y, points_z, **scatter_kwargs)
 
+    def fibonnaci_points(self, n_points=16000):
+        """Spherical Fibonacci point sets yield nearly uniform point
+        distributions on the unit sphere."""
+
+        x_vals = []
+        y_vals = []
+        z_vals = []
+
+        offset = 2. / n_points
+        increment = np.pi * (3. - np.sqrt(5.))
+
+        for i in range(n_points):
+            y = ((i * offset) - 1) + (offset / 2)
+            r = np.sqrt(1 - pow(y, 2))
+
+            phi = ((i + 1) % n_points) * increment
+
+            x = np.cos(phi) * r
+            z = np.sin(phi) * r
+
+            x_vals.append(x)
+            y_vals.append(y)
+            z_vals.append(z)
+
+        x_vals = [(self.radius * i) for i in x_vals]
+        y_vals = [(self.radius * i) for i in y_vals]
+        z_vals = [(self.radius * i) for i in z_vals]
+
+        return np.array([x_vals, y_vals, z_vals])
+
+    def plot_heatmap(self, ax, loss, n_points=16000, alpha=0.2, cmap='jet'):
```

Comment:
instead of `loss`, which is a vocabulary optimization's specific, could we use `scalar_field` or `scalar_function`? We might want to plot scalar fields on the sphere, and later on other manifolds, unrelated to any optimization problem.